config/vufind/permissions.ini

```diff
@@ -183,3 +183,8 @@ role = loggedin
 ;permission[] = "access.alma.webhook.challenge"
 ;require = ALL
 ;ipRange[] = "127.0.0.1"
+
+; Example permission for Notifications
+;[notifications.Admin]
+;permission[] = access.AdminModule
```

Comment:
I think this example should set `permission[]` to `notifications.Admin` or else it's just going to grant general admin privileges and not something more specific.

Reply:
I changed the permission needed to access the notifications settings to "notifications.Admin". Loggin in is required by default and an example how to add usernames is added. If the permission is not granted an error message will be displayed.